### PR TITLE
network/ipv6: add auto ipv6 generation type

### PR DESCRIPTION
**Pull request by pillip8282**
add auto ipv6 generation type

### CODE REVIEW

File: os/net/netmgr/netdev_lwip.c [hunk 1]
```diff
@@ -511,12 +511,27 @@ static int lwip_set_ip6addr(struct netdev *dev, struct sockaddr_storage *addr, i
 
 	return 0;
 }
+
+static int lwip_set_ip6addr_type(struct netdev *dev, uint8_t type)
+{
+	struct netif *ni = GET_NETIF_FROM_NETDEV(dev);
+	ni->ip6_addr_type = type;
```
Comment (jeongchanKim) [inline os/net/netmgr/netdev_lwip.c:518]: How about checking `ni` is NULL or not?

Comment (pillip8282) [reply os/net/netmgr/netdev_lwip.c:518]: every netdev contain netif. it's guaranteed.

File: os/net/lwip/src/core/netif.c [hunk 2]
```diff
@@ -1224,7 +1223,7 @@ err_t netif_gen_stable_private_id(struct netif *netif, s8_t addr_idx, ip6_addr_t
 		param.mac[i] = netif->hwaddr[i];
 	}
 	// ToDo : mbedTLS is in userspace. so mbedtls_sha256 can't be called.
-	//mbedtls_sha256(param.data, sizeof(param.data), rid.val, 0);
+	mbedtls_sha256(param.data, sizeof(param.data), rid.val, 0);
```
Comment (an4967) [inline os/net/lwip/src/core/netif.c:1226]: @pillip8282 
This can be issue when using loadable build type

**Review by sangwon03 — APPROVED** [review]
LGTM

**Pull request merged**